matching/matcher.go

```diff
@@ -131,3 +132,30 @@ func filterPlacesOnPriceLevel(places []Place, level POI.PriceLevel) []Place {
 	}
 	return results
 }
+
+// Filter parameters related with user rating
+type UserRatingFilterParams struct {
+	MinUserRatings int
+}
+
+// Matcher to filter places with low user ratings
+type MatcherForUserRatingsCount struct {
```

Comment:
My bad, maybe change the type name back to `MatcherForUserRatings`? Since we may add other parameters in the future such as the filter by rating itself.

Reply:
Done